easybuild/easyconfigs/s/SEED/SEED-2015.eb

```diff
@@ -0,0 +1,36 @@
+##
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::   Copyright 2014-2017 adelaide.edu.au/phoenix
+# Authors::	Robert Qiao <robert.qiao@adelaide.edu.au>, Exequiel Sepulveda <exequiel.sepulveda@gmail.com>
+# License::     Artistic-v2.0
+#
+# Notes::
+##
+
+easyblock = 'CmdCp'
+
+name = 'SEED'
+version = '2015'
```

Comment:
@robqiao please use `20150725` (i.e. the full datestamp of the commit) as version, and tie it to the commit ID:

```python
version = '20160725'
commit = 'f9784b0'
```

and then use the commit ID in the `sources` spec:

```python
sources = ['%s.tar.gz' % commit]